app/controllers/application_controller.rb

```diff
@@ -1,3 +1,4 @@
 class ApplicationController < ActionController::Base
-  protect_from_forgery with: :exception
+  include DeviseTokenAuth::Concerns::SetUserByToken
+  protect_from_forgery with: :null_session
```

Comment:
be careful with this null_session.

If you are creating an API and an application that renders html, you should not inherit both from the same ApplicationController, because the last one needs the protection_from_forgery